test/src/main/java/org/apache/accumulo/test/functional/BulkNewIT.java

```diff
@@ -186,18 +186,26 @@ public void testSingleTabletSingleFileOffline() throws Exception {
 
   @Test
   public void testMaxTablets() throws Exception {
-    String maxTablets = "0";
+    // test max tablets hit while inspecting bulk files
     try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
-      maxTablets = client.instanceOperations().getSystemConfiguration()
-          .get(Property.MASTER_BULK_MAX_TABLETS.getKey());
-      client.instanceOperations().setProperty(Property.MASTER_BULK_MAX_TABLETS.getKey(), "1");
+      tableName = "testMaxTablets_table1";
+      NewTableConfiguration newTableConf = new NewTableConfiguration();
+      // set logical time type so we can set time on bulk import
+      Map<String,String> props = new HashMap<>();
+      props.put(Property.TABLE_BULK_MAX_TABLETS.getKey(), "1");
+      newTableConf.setProperties(props);
+      client.tableOperations().create(tableName, newTableConf);
       testBulkFile(false, false);
-      fail("Expected IllegalArgumentException for " + Property.MASTER_BULK_MAX_TABLETS);
-    } catch (IllegalArgumentException e) {} finally {
-      try (AccumuloClient client = Accumulo.newClient().from(getClientProps()).build()) {
-        client.instanceOperations().setProperty(Property.MASTER_BULK_MAX_TABLETS.getKey(),
-            maxTablets);
-      }
+      fail("Expected IllegalArgumentException for " + Property.TABLE_BULK_MAX_TABLETS);
+    } catch (IllegalArgumentException e) {
+      // expected
+    }
+    // test max tablets hit using load plan
+    try {
+      testBulkFile(false, true);
+      fail("Expected IllegalArgumentException for " + Property.TABLE_BULK_MAX_TABLETS);
+    } catch (IllegalArgumentException e) {
+      // expected
     }
```

Comment:
I recently learned about this new feature in Junit.  It also returns the expected exception if you want to check the message.

```suggestion
   Assert.assertThrows(IllegalArgumentException.class, () -> testBulkFile(false, true));
```

Reply:
Cool thanks.  I am not sure what happened but Github isn't letting me apply your suggestion.  I will just change it manually.

Reply:
I settled with keeping the exception handling the way it is now.  I was able to clean up the IT using assertThrows and just checked the causes manually.  I think this is as clean as it is going to get until we move to Junit5.  I did find this library which I thought was interesting: https://assertj.github.io/doc/